scoper.inc.php

```diff
@@ -42,7 +42,7 @@
 
 return [
     'whitelist' => [
-        \Composer\Autoload\ClassLoader::class,
+        'Composer\*',
```

Comment:
that's crazy, I tried 

![image](https://user-images.githubusercontent.com/3725595/134062199-5138c17d-7155-4fdb-ae5c-5b164c520362.png)

and it didn't work, so I gave up :|

Reply:
Yeah, I tried that too, also some other random class and was prefixed. There is already [issue](https://github.com/humbug/php-scoper/issues/532) reported.